src/components/combobox/combobox.e2e.ts

```diff
@@ -810,6 +810,71 @@ describe("calcite-combobox", () => {
     });
   });
 
+  describe("calciteComboboxItemChange event correctly updates active item index", () => {
+    let page: E2EPage;
+    let element;
+    let lisboxItem;
+    let itemNestedLi;
+    let closeEvent;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(
+        html`
+          <calcite-combobox id="myCombobox">
+            <calcite-combobox-item value="Trees">
+              <calcite-combobox-item value="Pine">
+                <calcite-combobox-item id="PineNested" value="Pine Nested"></calcite-combobox-item>
+              </calcite-combobox-item>
+              <calcite-combobox-item value="Sequoia"></calcite-combobox-item>
+              <calcite-combobox-item value="Douglas Fir"></calcite-combobox-item>
+            </calcite-combobox-item>
+            <calcite-combobox-item value="Flowers">
+              <calcite-combobox-item value="Daffodil"></calcite-combobox-item>
+              <calcite-combobox-item value="Black Eyed Susan"></calcite-combobox-item>
+              <calcite-combobox-item value="Nasturtium"></calcite-combobox-item>
+            </calcite-combobox-item>
+          </calcite-combobox>
+        `
+      );
+      element = await page.find("calcite-combobox");
+      await element.click();
+
+      lisboxItem = await page.find("calcite-combobox-item#PineNested");
+      await lisboxItem.click();
+      await page.waitForChanges();
+
+      itemNestedLi = await page.find("calcite-combobox-item#PineNested >>> li");
+      closeEvent = page.waitForEvent("calciteComboboxClose");
+    });
+
+    it("clicking on Listbox item focuses on the item and closes out of Listbox with tab", async () => {
+      expect(itemNestedLi as any).toHaveClass("label--active");
```

Comment:
Typing the variables above should eliminate the need to use `any`, which we want to avoid whenever possible.

Reply:
Sidebar: it'd be good for reusability to start moving combobox's classes to a `CSS` object in the `resources` module.